get-started-with-tidb-lightning.md

```diff
@@ -20,21 +20,20 @@ TiDB Lightning 是一个将全量数据高速导入到 TiDB 集群的工具，
 
 ## 准备全量备份数据
 
-我们使用 [`mydumper`](/mydumper-overview.md) 从 MySQL 导出数据，如下：
+我们使用 [`dumpling`](/dumpling-overview.md) 从 MySQL 导出数据，如下：
 
 {{< copyable "shell-regular" >}}
 
 ```sh
-./bin/mydumper -h 127.0.0.1 -P 3306 -u root -t 16 -F 256 -B test -T t1,t2 --skip-tz-utc -o /data/my_database/
+./bin/dumpling -h 127.0.0.1 -P 3306 -u root -t 16 -F 256MB -B test -T test.t1,test.t2 -o /data/my_database/
 ```
 
 其中：
 
 - `-B test`：从 `test` 数据库导出。
-- `-T t1,t2`：只导出 `t1` 和 `t2` 这两个表。
+- `-T test.t1,test.t2`：只导出 `test.t1` 和 `test.t2` 这两个表。
```

Comment:
is it better to use `-f` here 🤔

Reply:
Any suggestion? `-f test.t[1-2]`? 🤔

Reply:
`-f test.t1 -f test.t2`

yeah you could write `-f 'test.t[12]'` if you want to be fancy 🤔